wsf_scraping/spiders/nice_spider.py

```diff
@@ -167,7 +195,7 @@ def save_pdf(self, response):
         is_pdf = response.headers.get('content-type', '') == b'application/pdf'
 
         if is_scraped(response.request.url):
-            self.logger.info(
+            self.logger.debug(
```

Comment:
why did you change that?

Reply:
It created a lot of noise in the logs, for the default log level is INFO. Also, it seems more of a DEBUG level information, as it only logs when we already did download the pdf.